src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx

```diff
@@ -191,8 +219,8 @@ export default class ColumnCustomizationContent extends Vue {
         </div>
         <div class={UTILITY_CLASSES.FLEX.FLEX_GROW1}>
           <SelectedColumns
-            locked-columns={this._selectedLockedColumns}
             standard-columns={this._selectedStandardColumns}
+            handle-select-column={this._handleSelectColumn}
```

Comment:
Remember to not pass function handlers as parameters. Emit events from the descendant component instead and set your handler in an eventListener.